scripts/sem_bleaching_map.py

```diff
@@ -80,31 +104,36 @@ def __init__(self, dt, roi, pxs, subpx=1, lpower=0.2):
         self.ccd = model.getComponent(role="ccd")
         self.stage = model.getComponent(role="stage")
 
-        # Find the index with maximum power to use to set lpower
-        self.max_power_index = list(self.light.power.range[1]).index(max(self.light.power.range[1]))
+        self.ccd_acq_complete = threading.Event()
+        self._spotsimage = None
+
+        # scale is the distance between the SR pixels in SEM pixels at ebeam.scale=1
+        self.scale = (pxs / self.ebeam.pixelSize.value[0], pxs / self.ebeam.pixelSize.value[1])
+
+        self.power_index = lemissions
         lprng = self.light.power.range
-        if not (lprng[0][self.max_power_index] < lpower < lprng[1][self.max_power_index]):
+        if not (lprng[0][self.power_index] < lpower < lprng[1][self.power_index]):
```

Comment:
Here you are excluding `lpower` from being either 0 or range maximum. Is that intended or these values can be added ( ie replace < with <=  ) ?